roles/common/templates/monitoring/sensu-client-path

```diff
@@ -0,0 +1,5 @@
+{% if "ceph_monitors" not in group_names -%}
+export PATH="{{ common.python.base_venv }}/bin:/opt/sensu/embedded/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/etc/sensu/plugins"
+{% else %}
+export PATH="/opt/sensu/embedded/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/etc/sensu/plugins"
```

Comment:
mimic what cinder and glance did to create link for rados and rdo packages.
find another package missing:
(base)[root@cpm1 site-packages]# ceph health
Traceback (most recent call last):
  File "/bin/ceph", line 124, in <module>
    from ceph_argparse import \
ImportError: No module named ceph_argparse